joomla/joomla-cms
# [4.3] Update deleted files list in script.php for upcoming 4.3.0-alpha2

Pull Request for Issue # .

### Summary of Changes

1. Update the section comments for deleted files and folders for 4.3 and to the current 4.3.0-alpha1.

2. Update deleted files to changes from following PRs:
- #38466 - file `/plugins/content/emailcloak/emailcloak.php`
- #38470 - file `/plugins/content/fields/fields.php`

3. Remove js files from PR #38823 from the deleted files list because we have to keep them for b/c of layout overrides.
For the same reason, the deleted js files from PR #39374 are not added to the list with change 2. above.
The broken build will be fixed with another PR, see discussion in my PR's #39408 and #39413 .

### Testing Instructions

Code review.

Or if you want to make a real test, update a 4.3.0-alpha1 or any older 4.x version to the last 4.3 nightly build to get the actual result, and update a 4.3.0-alpha1 or any older 4.x version to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

Files and folders mentioned at item 2 of the description above are still present after updating from a 4.3.0-alpha1 or any older 4.x version.

### Expected result AFTER applying this Pull Request

All files and folders mentioned at item 2 of the description above have been deleted after updating from a 4.3.0-alpha1 or any older 4.x version.

### Link to documentations
Please select:
